test/unit/models/original_test.rb

```diff
@@ -18,13 +18,13 @@
 
 class OriginalTest < ActiveSupport::TestCase
   test "FactoryGirl works for noise's original" do
-    assert_difference('Original.count') do
+    assert_difference ->{Original.count} do
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.